ansible/group_vars/bananas_fileservers

```diff
@@ -0,0 +1,136 @@
+# Our vars, for easy reuse
+cache_timeout: "1y"  # Basically just as long as nginx runs for
+bananas_production_cdn: bananas.cdn.openttd.org
+bananas_staging_cdn: bananas.cdn.staging.openttd.org
+staging_hostname: "{{ inventory_hostname_short }}.cdn.staging.openttd.org"
+
+certbot_admin_email: info@openttd.org
+certbot_create_if_missing: true
+certbot_create_standalone_stop_services: []
+certbot_certs:
+  - domains:
+      - "{{ inventory_hostname }}" # Note that this is the "primary" and contains the certificate for the staging domain
+      - "{{ staging_hostname }}"
+
+
+nginx_remove_default_vhost: true
+nginx_extra_http_options: |
+  proxy_cache_path /tmp/nginx-cache/ levels=1:2 keys_zone=nginx-cache:16m max_size=1g inactive={{ cache_timeout }} use_temp_path=off;
```

Comment:
We need 2 caches, 1 for production, 1 for staging. Otherwise you can pollute one via the other.

Reply:
mm, I thought so. But how to do that? this proxy path is a "global" variable

Reply:
It isn't :) `keys_zone=nginx-cache`, `nginx-cache` is the name of the cache. This is later used with `proxy_cache nginx-cache`. So if you add another one, say, `nginx-staging-cache`, it will work fine :)